### PR TITLE
make k8s deployment script executed

### PR DESCRIPTION
**Pull request by xunpan**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/cloud/ibmcloud/actuators/machine/actuator.go [hunk 1]
```diff
@@ -161,7 +162,8 @@ func (ic *IbmCloudClient) Delete(ctx context.Context, cluster *clusterv1.Cluster
 // Update updates a machine and is invoked by the Machine Controller
 func (ic *IbmCloudClient) Update(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	log.Printf("Updating machine %v for cluster %v.", machine.Name, cluster.Name)
```
Comment (gyliu513) [inline pkg/cloud/ibmcloud/actuators/machine/actuator.go]: klog here?

File: cmd/clusterctl/configuration/ibmcloud/provider-component/user-data/ubuntu/templates/master-user-data.sh [hunk 2]
```diff
@@ -92,9 +92,6 @@ cat > /etc/systemd/system/kubelet.service.d/20-kubenet.conf <<EOF
 Environment="KUBELET_DNS_ARGS=--cluster-dns=${CLUSTER_DNS_SERVER} --cluster-domain=${CLUSTER_DNS_DOMAIN}"
 EOF
 
-echo $OPENSTACK_CLOUD_PROVIDER_CONF | base64 -d > /etc/kubernetes/cloud.conf
```
Comment (gyliu513) [inline cmd/clusterctl/configuration/ibmcloud/provider-component/user-data/ubuntu/templates/master-user-data.sh]: Do we need this for ibmcloud?

File: pkg/cloud/ibmcloud/actuators/machine/actuator.go (hunk 1 above)
Comment (xunpan) [reply pkg/cloud/ibmcloud/actuators/machine/actuator.go]: I did not change all parts from `log` to `klog`. So, I just change on demand for this ticket. 
A ticket for change `log` to `klog` was logged for this issue.

File: cmd/clusterctl/configuration/ibmcloud/provider-component/user-data/ubuntu/templates/master-user-data.sh (hunk 2 above)
Comment (xunpan) [reply cmd/clusterctl/configuration/ibmcloud/provider-component/user-data/ubuntu/templates/master-user-data.sh]: until now, we do not need this `cloud.yaml`. We can access IBM cloud by `userName` and `apiKey`.

**Comment by xunpan** [comment]
master node deployed. I think we can merge it if there is no other comments.

I'm working on slave node deployment

**Pull request merged**